### PR TITLE
Implement Uptime Webhook

### PR DESCRIPTION
**Pull request by Salanto**

### CODE REVIEW

File: core/src/discord.cpp [hunk 1]
```diff
@@ -56,6 +80,21 @@ QJsonDocument Discord::constructModcallJson(const QString &f_name, const QString
...
 }
 
+QJsonDocument Discord::constructUptimeJson(const QString& f_timeExpired)
+{
+    QJsonObject l_json;
+    QJsonArray l_array;
+    QJsonObject l_object {
+        {"color", "13312842"},
+        {"title", "Your server is still running!"},
+        {"description", "Hello World!\nYour server has been online for " + f_timeExpired}
```
Comment (in1tiate) [inline core/src/discord.cpp:90]:
```suggestion
        {"description", "Hello, World!\nYour server has been online for " + f_timeExpired}
```

**Review by in1tiate — APPROVED** [review]
if it fits i must acquit

**Comment by in1tiate** [comment]
@scatterflower give this one a pass when you get the chance please

**Comment by scatterflower** [comment]
please squash these commits, and stop using terrible commit names

**Comment by in1tiate** [comment]
@Salanto resolve the merge conflicts and I'll merge this

**Pull request merged**